src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -62,7 +62,7 @@
  * |-----------------------------------------------------------|----------------------------------------------------------------------------|
  * | Find any tag.                                             | `$tags->next_tag();`                                                       |
```

Comment:
Lines 61-63 need a little alignment fixing per @dmsnell's comment. Is this something that could be fixed ahead of a commit?

Reply:
Sure it can be fixed.

Reply:
Aligned in https://github.com/WordPress/wordpress-develop/pull/4168/commits/94b8f1fefe13c875509066859461d56c3a58a03d @costdev @dmsnell